CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Coding guidelines
```

Comment:
I think we could use a Feature Request section as well, if it's any different than the Bug Reports section. Having a Pull Request section also seems useful (we could explain the labelling and the code review groups). There's also documentation for mudlet though I don't even know how to edit it, so that section seems really relevant. Build and Test sections should hopefully become relevant once we have more testing. :)

I'm not going to suggest that we have all of these right now, nor would I suggest just copying the Rust one. For example, we could have sections for Lua vs C++ contributions.

As for the more general comment about the scale of Rust vs Mudlet, I agree that the word scale (meaning number of contributors, etc) would have them be different. But the implication of the rest of your comment is that Rust targets a different set of developers when getting contributions and that simply isn't true. Their goal is to get contributions from a wide variety of people with all sorts of backgrounds. Their primary documentation writer is a former ruby on rails programmer, for example, not a C++ expert. Just because I'm a C++ expert doesn't mean that everything I advocate for is to benefit only C++ experts. I'd prefer if my suggestions not meet that particular objection over and over.